packages/block-library/src/buttons/content-justification-dropdown.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * WordPress dependencies
+ */
+import { DropdownMenu } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	contentJustificationCenterIcon,
+	contentJustificationLeftIcon,
+	contentJustificationRightIcon,
+} from './icons';
+
+const DEFAULT_ALLOWED_VALUES = [ 'left', 'center', 'right' ];
+
+const CONTROLS = {
+	left: {
+		icon: contentJustificationLeftIcon,
+		title: __( 'Justify items left' ),
+	},
+	center: {
+		icon: contentJustificationCenterIcon,
+		title: __( 'Justify items center' ),
+	},
+	right: {
+		icon: contentJustificationRightIcon,
+		title: __( 'Justify items right' ),
+	},
```

Comment:
The three items still have the title 'items' instead of 'content'

Reply:
Oh, whoops! Thanks for catching that. Fixed.